package.json

```diff
@@ -254,17 +254,17 @@
         },
         {
           "command": "vscode-cmsis-debugger.liveWatch.modify",
-          "when": "view == cmsis-debugger.liveWatch && viewItem == expression",
+          "when": "view == cmsis-debugger.liveWatch",
```

Comment:
This one should also be only be available for `parentExpression`. It's about modifying an expression, not a value. Hence not possible for children of the top-level expression.